About/about.html

```diff
@@ -0,0 +1,152 @@
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="mobileNav d-md-none">
+        <i class="fa fa-bars pt-4 fa-3x" id="hamburger" aria-hidden="true"></i>
+        <img id="closeX" class="d-none" src="../img/close.png" alt="" />
+      </nav>
+      <nav class="desktopNav mt-3">
+        <div
+          class="destopNavItems d-flex align-items-center justify-content-between"
+        >
+          <div class="logo ms-5 d-none d-md-block">
+            <a href="../Home/index.html"
+              ><img src="../img/logo.jpeg" alt="logo"
+            /></a>
+          </div>
+          <ul
+            id="desktopList"
+            class="items list-unstyled d-flex gap-4 align-items-center d-none d-md-flex"
+          >
+            <li>
+              <a class="text-decoration-none text-dark" href="#event"
+                >Recent Event</a
+              >
+            </li>
+            <li>
+              <a class="text-decoration-none text-dark" href="#sponsors"
+                >Sponsors</a
+              >
+            </li>
+            <li class="ms-5 me-5 border border-danger p-2 bg-danger">
+              <a
+                class="text-decoration-none text-white"
+                href="../Home/index.html"
+                >Home</a
+              >
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <h3 class="text-center">"Sharing with the world"</h3>
+      <h1 class="text-center col-md-6 mx-auto">
+        THE PERFECT BLUEPRINT TO E-COMMERCE 2022
+      </h1>
+    </header>
+
+    <main class="aboutMain col-md-12">
+      <div class="aboutHero hero">
+        <p class="text-center bg-white col-md-6 mx-auto p-3">
+          The eye-opening e-comm whitepaper brings together the community of
+          experts and academics who compromise the creative affiliate network in
+          different countries every two years. <br />
+          This year we hope to expand our invitation list including
+          organizations and individuals
+        </p>
+        <p class="text-center pb-5 pt-3 col-md-6 mx-auto">
+          Kindly contact us via email for any further question about OXJAYENT
+          E-COMM WHITEPAPER 2022 <br />
+          support@oxjayent.com
+        </p>
+      </div>
+
+      <section class="aboutProgram mainProgram text-white">
+        <div
+          class="Program forum d-flex align-items-center d-md-flex justify-content-center gap-3"
+        >
+          <img src="../img/icons-3.svg" alt="" />
+          <div>
+            <h2>E_COMM the future of Marketing</h2>
+            <p>
+              It's imposible to overstate the importance of e-commerce in the
+              world of today. If you ever bought things and delivered to you
+              online, it is the best experience in shopping.
+            </p>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] As I can see here the section is very diffrenet than the original section so kindly update it and make the order of the elements the same so first you have a heading then a paragraph then your logo you can use flexbox to align that 💪🏻✅✅

- [ ] Kindly use the same background color as the oriignal design so to be white in this case since it is important to not change in the design just the texts and images ✅✅✅

- Your design
![image](https://user-images.githubusercontent.com/58553711/181144107-799eeb97-e6c9-4462-b2de-af008c51a9f4.png)


- Original design
![image](https://user-images.githubusercontent.com/58553711/181143872-b6ed17ca-54a7-4b45-b931-b17065b817ed.png)

Reply:
Section changed